src/SQLQueryStress/Form1.cs

```diff
@@ -472,13 +472,25 @@ private void UpdateUi()
 
         private void btnCleanBuffer_Click(object sender, EventArgs e)
         {
+            if (!_settings.MainDbConnectionInfo.TestConnection())
+            {
+                MessageBox.Show(Resources.MustSetValidDbConnInfo);
+                return;
+            }
+
             MessageBox.Show(LoadEngine.ExecuteCommand(_settings.MainDbConnectionInfo.ConnectionString, "DBCC DROPCLEANBUFFERS")
                 ? "Buffers cleared"
                 : "Errors encountered");
         }
 
         private void btnFreeCache_Click(object sender, EventArgs e)
         {
+            if (!_settings.MainDbConnectionInfo.TestConnection())

```

Comment:
Already in master branch

Reply:
Rebased my fix-warnings branch on my master now.